### ISSUE TITLE
The propensity values are not in range [-1,1]

### ISSUE THREAD

**Issue by ghost** [PeptoneInc/dspp-keras#24]
Hi,
As I understood from your `README.md` the propensity values should be between -1 and 1, which is not fulfilled in your dataset when I use dspp-keras package.

Could you please explain what is the issue here?

I put my example here.
```
x,y=dspp.load_data()
y[0].max()

```
which results in `2.5`.

Could you please explain how you normalize your dataset? 
by min-max scaler? normal-distribution scaler?

**Comment by jandom** [comment]
@ktamiola should probably tackle this one. 

briefly here from me: we have shifted the original scale from -1,1 by +2, so it's now 1 for beta sheet, 2 for random-coil and 3 for helix. 0 now indicates no data for a given sequence position.

@ErmiaAzarkhalili many thank for reporting this inconsistency, you're obviously correct, we'll get it sorted asap

**Comment by ktamiola** [comment]
Dear @ErmiaAzarkhalili , indeed I have mistakenly used in our `README.md` an old functional propensity scale. We have shifted all the data by `2.0` as Keras and Tensorflow can't handle `np.nan` in loss and gradient computation procedures. Thank you for reporting this serious discrepancy.

**Comment by ghost** [comment]
@ktamiola @jandom Thanks for the quick update. could you please update the `README.md` file?

**Comment by ktamiola** [comment]
Absolutely! The `README.md` file has been updated already!
